src/app/assessments/assessments-dashboard/assessments-dashboard.component.ts

```diff
@@ -88,6 +89,64 @@ export class AssessmentsDashboardComponent implements OnInit {
             );
     }
 
+    public populatePhaseRiskTree(phase: any) {
+        const riskTree = {};
+
+        // Assessed Objects per phase
+        if (phase !== undefined && phase.assessedObjects !== undefined) {
+            phase.assessedObjects.forEach((assessedObject) => {
+                // Questions per assessed object
+                assessedObject.questions.forEach((question) => {
```

Comment:
is there ever a concern if assessedObject.questions is undefined?

Reply:
That's a good catch. It's unlikely for it to happen in practice, but a custom made object could trigger an issue!
I'll fix it.

Reply:
Fix via: https://github.com/unfetter-discover/unfetter/issues/646